### DISCUSSION TITLE
Can't setup Prisma in my turbo repo

### DISCUSSION THREAD

**Discussion by Manohar-Penta** [vercel/turborepo#9116] [Help]
### Summary

Followed documentation for setting up Prisma with turborepo and this is the error I encountered.

![Screenshot 2024-09-06 154450](https://github.com/user-attachments/assets/b1c94324-c32e-4efe-a6e6-2b01bc99cb71)

### Additional information

I am setting up a project and wanted to use prisma. I have followed [this article](https://turbo.build/repo/docs/guides/tools/prisma#figuring-out-the-scripts) to setup.

After following everything on the article, I got a error 
"Package @prisma/client (serverComponentsExtenalPackages or default list) can't be external"

I did the same setup few months back and it worked perfectly. I don't know why this is happening now.

**Comment by petertjmills** [comment]
Getting the same issue. Seeing that this is closed. Was there a resolution?

**Reply by gempi** [reply]
I run into this issue with pnpm. Just had to set `public-hoist-pattern[]=*prisma*` in my .npmrc file, remove all `node_modules` and install the packages again.

**Reply by chinanderm** [reply]
Thank you @gempi! This helped me a lot getting Prisma working within a pnpm monorepo as a package. The Prisma client is getting shared out to two Remix apps and a NodeJS app built with tsup.
